config/deploy/olm/catalogsource.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: limitador-operator-catalog
+spec:
+  sourceType: grpc
+  image: quay.io/kuadrant/limitador-operator-catalog:eguzki-45
```

Comment:
Probably left after testing, maybe the default config be pointing to `latest`

Reply:
good catch!